### DISCUSSION TITLE
How can I still display global navbar when I use a custom layout?

### DISCUSSION THREAD

**Discussion by yihui-dev** [vuepress/core#1320] [Q&A] [answered]
I am using a custom layout named **Fullwidth**, Here is my index.md:

```md
---
layout: FullWidth
---

my content
```

And I create the Fullwidth.vue:

```vue
<template>
  <Content />
</template>

<script setup></script>

<style lang="scss" scoped></style>

```

Finally，I config the Fullwidth.vue in `.vuepresss/client.js`:

```js
export default defineClientConfig({
  layouts: {
    FullWidth,
  }
})
```

But when I visited the page, I found that the entire page only showed **"my content"** without the header navbar. I expected the navbar to still be displayed. What should I do to keep the navbar visible?

**Comment by Mister-Hope** [comment, marked as answer]
With slots, https://vuejs.press/advanced/cookbook/making-a-theme-extendable.html
